pkg/users/accounts-list.js

```diff
@@ -38,11 +40,36 @@ import { SearchIcon } from '@patternfly/react-icons';
 import { SortByDirection } from "@patternfly/react-table";
 import { account_create_dialog } from "./account-create-dialog.js";
 import { delete_account_dialog } from "./delete-account-dialog.js";
+import { delete_group_dialog } from "./delete-group-dialog.js";
 import { lockAccountDialog } from "./lock-account-dialog.js";
 import { logoutAccountDialog } from "./logout-account-dialog.js";
 
 const _ = cockpit.gettext;
 
+const GroupActions = ({ group, accounts }) => {
+    const [isKebabOpen, setKebabOpen] = useState(false);
+
+    if (!superuser.allowed)
+        return null;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17996-20221206-122041-ea567d53-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)